### DISCUSSION TITLE
通知栏与应用界面发生干涉

### DISCUSSION THREAD

**Discussion by HofNature** [xiaoyaocz/flutter_dmzj#137] [General]
有概率会发生应用界面未规避通知栏的情况，即通知栏会覆盖在应用界面之上。
触发条件未知。
![Screenshot_2023-03-21-02-37-47-341_com.xycz.dmzjx.jpg](https://user-images.githubusercontent.com/35188269/226435442-c9b4af63-f9cf-45da-875b-89f44abc4569.jpg)

**Comment by xiaoyaocz** [comment]
#139
